Lib/platform.py

```diff
@@ -31,6 +31,7 @@
 #
 #    <see CVS and SVN checkin messages for history>
 #
+#    1.0.9 - added cache_clear() function to invalidate cached values
```

Comment:
Typo?
```suggestion
#    1.0.9 - added invalidate_caches() function to invalidate cached values
```

Reply:
Yes, in the first version, I used `cache_clear()` like for `lru_cache`d objects, but `invalidate_caches` is used by `importlib`. I preferred the second one since it's clearing global objects and not clearing objects attached to functions themselves (like `lru_cache.cache_clear` does). Well... `filecmp` uses `clear_cache()`, `linecache` uses.... `clearcache()` so at this point I'm just picking the one I prefer.

Reply:
It might be worth making another issue to "standardize" the name for a cache-clear function (or probably, a thread on discourse). `cache_clear` could become a deprecated alias to `invalidate_caches` and whatnot.

Reply:
Having `clear_cache()` and `clearcache()` is definitely a bad idea because you need to remember two different names (though I doubt anyone is using linecache and filecmp a lot...). Note that `clear_cache` is also used by `urllib.parse`.

`invalidate_caches` is used by import-based modules such as `importlib` and `zipimport` (and now `platform`). And `cache_clear()` is used by `functools.lru_cache`. 

I would say that `invalidate_caches` is the most expressive one, followed by `clear_cache()`, then by `cache_clear()`. But in practice, I think people want to minimize the number of key strokes, so I would say that `clearcache()` is the most efficient one in that sense (but also the less convenient to read).

You can post something on discourse if you want to normalize the function used for invalidating caches in general, but I don't think it's of high priority.